setup.py

```diff
@@ -82,10 +83,10 @@ def read(*filenames, **kwargs):
         "Intended Audience :: Science/Research",
         "License :: Free for non-commercial use",
         "Operating System :: OS Independent",
-        "Programming Language :: Python :: 2.7",
         "Programming Language :: Python :: 3.4",
         "Programming Language :: Python :: 3.5",
         "Programming Language :: Python :: 3.6",
+        "Programming Language :: Python :: 3.7",
```

Comment:
I guess before #1514 gets merged in we should decide if we are making 3.7 support official? Over there I was assuming not, but we are here.

Reply:
After some offline discussion, my mistake was that I had been defining "support" as testing against a clean/reproducible environment. But saying that "we support Python 3.7" does not mean that we are necessarily testing against a clean/reproducible Python 3.7 environment. Rather, the consensus was that the testing within the release process on a developer's py3.7 environment is sufficient and that official "support" means we will simply try to fix any bugs that emerge on py3.7 environments.